src/lib/validate-forms.js

```diff
@@ -0,0 +1,67 @@
+const api = require('./api')();
+const argsFormFilter = require('./args-form-filter');
+const fs = require('./sync-fs');
+const log = require('./log');
+const { getFormDir, getFormFilePaths } = require('./forms-utils');
+
+//TODO Add this to the main loop
+module.exports = (projectDir, subDirectory, options) => {
+  if (!options) options = {};
+
+  const formsDir = getFormDir(projectDir, subDirectory);
+  if(!formsDir) {
+    log.warn(`Forms dir not found: ${formsDir}`);
```

Comment:
I don't think we need to warn. It may be on purpose, eg: an SMS project with no Enketo forms.